### ISSUE TITLE
Condition_scope will be included all the time you run terraform plan

### ISSUE THREAD

**Issue by MadhuraDhanajaya** [newrelic/terraform-provider-newrelic#770]
### Terraform Version
Terraform version = 0.12.24
Provider version = 1.16.0

### Affected Resource(s)
**newrelic_alert_condition**

### Terraform Configuration Files
```
resource "newrelic_alert_condition" "alert_conditions" {
  name     = condition_name
  type     =  each.value["metric_type"]
  entities = [local.entity_ids_by_service_names[each.value["service"]]]
  metric   = each.value["metric"]

  condition_scope = "instance"

  gc_metric             = each.value["gc_metric"]
  policy_id             =  policy_id 
  each.value["metric_type"])]
  violation_close_timer = each.value["violation_close_timer"]

  term {
    operator          = above
    threshold         = 95
    priority            = critical
    time_function  = all
    duration          = 5 
  }

```

### Actual Behaviour
  ``` 
module.alert_policies.newrelic_alert_condition.alert_conditions["application_name_heap_memory_usage"] will be updated in-place
  ~ resource "newrelic_alert_condition" "alert_conditions" {
      + condition_scope       = "instance"
        enabled               = true
        entities              = [
           122222,
        ]
        id                    = "*****"
        metric                = "heap_memory_usage"
        name                  = "abc_heap_memory_usage"
        policy_id             = ****
        type                  = "apm_jvm_metric"
        violation_close_timer = 24

        term {
            duration      = 5
            operator      = "above"
            priority      = "critical"
            threshold     = 95
            time_function = "all"
        }
    }

Plan: 0 to add, 478 to change, 0 to destroy.
```
### Expected Behavior
On running terraform plan or terraform apply only newly added resource or change in any resource has to be modified but here the **condition scope** which is set to **instance**  will get updated all the time we run `plan` and `apply` which has no changes in it. 

Result should be  **Plan: 0 to add, 0 to modify, 0 to destroy.**  

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`
2. `terraform apply`

**Comment by MadhuraDhanajaya** [comment]
Any updates on this?

**Comment by sanderblue** [comment]
Hi @MadhuraDhanajaya, I've tried to reproduce this but so far I haven't been able to yet. Can you share an example config without variables or loops so we have a more specific scenario to debug? If we could scope to a single alert condition it might help with debugging. 

I also noticed that you're currently on version 1.16.0 of the provider. Could you try this on the latest 1.x version (1.19.1) of the provider to see if it's still an issue on 1.19.1?

**Comment by sanderblue** [comment]
Quick update. I'm able to reproduce this now. Will report back with a diagnosis in a bit.

**Comment by sanderblue** [comment]
After debugging for a bit, it looks like the REST API does not return `condition_scope` for alert conditions of the type `apm_jvm_metric`. This type of condition is created in the UI via the "JVM Health Metric" category and is automatically scoped to the individual JVM instances, hence the reason the `condition_scope` property isn't returned.

TL;DR you should be able to remove the `condition_scope` attribute for alert conditions that are type `apm_jvm_metric` since these are automatically scoped to JVM instances.

**Comment by MadhuraDhanajaya** [comment]
Thanks for the update @sanderblue

**Comment by sanderblue** [comment]
Closing per [solution mentioned earlier](https://github.com/newrelic/terraform-provider-newrelic/issues/770#issuecomment-659583861) 🙂
